plugin/federation/federation.go

```diff
@@ -132,22 +132,33 @@ type Entity {
 `
 	}
 
-	return &ast.Source{
-		Name:    "federation/entity.graphql",
-		BuiltIn: true,
-		Input: `
-# a union of all types that use the @key directive
-union _Entity = ` + entities + `
-` + resolvers + `
+	_serviceTypeDef := `
 type _Service {
   sdl: String
-}
+}`
+	additionalQueryFields := ``
```

Comment:
```suggestion
	var additionalQueryFields string
```

Reply:
Hi, I have adjusted the code, please review it again at your convenience, thank you.